pkg/filepromoter/file.go

```diff
@@ -119,10 +138,20 @@ func (o *copyFileOp) Run(ctx context.Context) error {
 func (o *copyFileOp) String() string {
 	return fmt.Sprintf(
 		"COPY %q to %q",
-		o.Source.AbsolutePath, o.Dest.AbsolutePath)
+		o.Source.Path(), o.Dest.AbsolutePath)
 }
 
-func computeSHA256ForFile(filename string) (string, error) {
+func ComputeSHA256ForFile(filename string) (string, error) {
+	hasher := sha256.New()
+	return computeHashForFile(filename, hasher)
+}
+
+func ComputeMD5ForFile(filename string) (string, error) {
+	hasher := md5.New()
```

Comment:
Nit: If md5.New() never fails and always returns a hasher, you might as well just pass that into the invocation at 151 and do away with this 'hasher' var.

But at that point, I think it would be reasonable to just use `computeHashForFile` directly and do away with these wrapper functions. /shrug